index.html

```diff
@@ -84,6 +84,7 @@ <h1 id="title">Survey Exercice</h1>
     - The freeCodeCamp Team
 
     -->
+    .
```

Comment:
You should delete this comment from freeCodeCamp so your code looks cleaner.

Reply:
- [ ] Kindly always include an explanation, or a "why", for each requested change, this will make your comments more useful and clear

Reply:
You should delete this comment from freeCodeCamp so your code looks cleaner, in that way your work is going to look more professional. Details are always important! Keep going!